README.md

```diff
@@ -324,8 +324,8 @@ interface Observable {
   Observable take(unsigned long long);
   Observable drop(unsigned long long);
   Observable flatMap(Function project);
-  Observable toArray();
-  Observable forEach(Function callback);
+  Promise<sequence<any>> toArray();
```

Comment:
Suggestion: I'd almost recommend we drop this entirely for now. It could be added in a follow up.

Reply:
Let's consider that outside of this PR. I think initially, it would be nice to match iterator helpers as closely as possible, and `toArray()` is one of those helpers, so if we remove it we should do so with careful public notice that we're deviating.

Reply:
Ah! I had missed it was one of those helpers. I often don't like how it's used in RxJS. Haha.